tests/components/application-connector/test/application-gateway/manual_test.go

```diff
@@ -0,0 +1,27 @@
+package application_gateway
+
+import (
+	"net/http"
+	"time"
+)
+
+func (gs *GatewaySuite) TestManual() {
+	gs.Run("OAuth token renewal", func() {
+		url := gatewayURL("manual", "oauth-short") + "/ok"
```

Comment:
I think we can move the `+ "/ok"` part to the Application CR

Reply:
I am guessing different endpoints might be needed for different manual tests, but we can change it when we get there and leave it precise (with `/ok` endpoint) for now